superset-frontend/plugins/plugin-chart-pivot-table/src/react-pivottable/TableRenderers.jsx

```diff
@@ -47,7 +48,7 @@ function displayHeaderCell(
       >
         {ArrowIcon}
       </span>
-      <span className="toggle-val">{parseLabel(name)}</span>
+      <span className="toggle-val">{t(parseLabel(name))}</span>
```

Comment:
Hello @villebro @aehanno!
We should not translate the results of the parseLabel function, because users can just set the verbose name of the column/metric in the dataset or manual SQL, excluding the string 'metric' that appears in the top left corner.
![image](https://user-images.githubusercontent.com/66589759/214359754-b5c1a28c-2084-4f41-a57e-f83cefc62765.png)

To translate that metric, we can just add a translation for the METRIC_KEY constant at superset-frontend/plugins/plugin-chart-pivot-table/src/PivotTableChart.tsx:59, and it will work properly.
Result of translating METRIC_KEY:
![image](https://user-images.githubusercontent.com/66589759/214359989-68ed7755-046d-4dfb-b6c6-d5a2793355dc.png)

Reply:
I just do the correction, thanks